src/components/decks/DeckListing.vue

```diff
@@ -140,6 +145,15 @@ export default {
         this.filterList(() => {})
       }
     )
+    watch(
+      [
+        () => this.preconOnly,
```

Comment:
Since the logic is identical, you should be able to include this in the previous `watch` function call like:

```js
watch(
  [
    () => this.phoenixborn,
    () => this.preconOnly,
  ]
)
```

Reply:
Sure thing! I'll get both of these up in just a bit

Reply:
@onecrayon Changes are pushed up, condensed them into one watch function, and I made sure when the values are changed it calls filterList as it's supposed to